gremlin_python/structure/io/graphson.py

```diff
@@ -67,8 +69,10 @@ def _dictify(object):
 
     @staticmethod
     def writeObject(objectData):
-        return json.dumps(GraphSONWriter._dictify(objectData), separators=(',', ':'))
-
+        if use_separators:
+            return json.dumps(GraphSONWriter._dictify(objectData), separators=(',', ':'))
+        else:
```

Comment:
Maybe we should just remove the separators kwarg because it isn't really necessary.